tests.py

```diff
@@ -311,6 +311,28 @@ def test_import_from_qtwidgets():
     assert QPushButton.__name__ == "QPushButton", QPushButton
 
 
+def test_pyside2uic():
```

Comment:
Ok, I'll just remove the test then. And we should be good to go and merge this if you feel it looks allright.